docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -254,3 +254,43 @@ Note that use of the "add_host" module above creates a temporary, in-memory grou
 in the 'new_instances' group, if so desired.  Any sort of arbitrary configuration is possible at this point.
 
 For more information about Google Cloud, please visit the `Google Cloud website <https://cloud.google.com>`_.
+
+Migration Guides
+----------------
+
+gce.py -> gcp_compute_instance.py
+`````````````````````````````````
+We're encouraging everyone to move from the `gce` module to the
+`gcp_compute_instance` module. The new `gcp_compute_instance` module has better
+support for all of GCP's features, less dependencies, more flexibility, and
+better supports GCP's authentication systems.
+
+The new `gcp_compute_instance` module supports all of the features of the `gce`
+module (and more!). Below is a mapping of `gce` fields over to
+`gcp_compute_instance` fields.
+
```

Comment:
The parameter mapping is helpful. An example playbook shown "before" and "after" would be a great addition.

Reply:
I put in an example playbook which shows the loop concept. I think that's the hardest part of the transition. It doesn't show before/after as much because the parameter mapping shows all of the possible transitions in my opinion.